pkgs/getsops/sops/registry.yaml

```diff
@@ -1,13 +1,13 @@
 packages:
   - type: github_release
-    repo_owner: mozilla
+    repo_owner: getsops
     repo_name: sops
     description: Simple and flexible tool for managing secrets
     format: raw
     overrides:
       - goos: windows
         asset: sops-{{.Version}}.exe
-    version_constraint: semver(">= 3.7.2")
```

Comment:
Why do you fix the version constraint?
I don't think we need to fix the version constraint.